### PR TITLE
add flowchart (fixes #118)

### LINKED ISSUE #118: add flowchart
In step 6 "GitHub Issues" there is a missing flowchart diagram. I suggest adding a diagram in place of the "NEED DIAGRAM HERE"  text (see picture below)
image

here is the flowchart I have drafted:
image

### PR DESCRIPTION
**Pull request by MatthewMarroquin**
https://raw.githack.com/Mattmtech/Mattmtechtreehouses.github.io/add-flowchart/#!./pages/vi/github-issues.md

[x] Check for issue number in pull request title
[x] Are there any unneeded files in the pull request?
[x] Did they make a branch for their patch?
[x] Does the pull request actually fix the issue?
[x] Check the pull request on raw.githack, does it display without any errors?
[x] Are there any merge conflicts?
[x] Make sure that people use their GitHub accounts when making commits through git

### CODE REVIEW

File: pages/vi/github-issues.md [hunk 1]
```diff
@@ -65,13 +65,13 @@ Now you can go and make the proposed changes to your local files. You can use an
 | --- |
 If you're still confused, that's quite alright. [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow) can be quite challenging at first. See this diagram below. For better context, the new branch you just made is inside your "Forked GitHub IO".
 
-## NEED DIAGRAM HERE
+![flowchart](/images/flowchart.png)
```
Comment (lmmrssa) [inline pages/vi/github-issues.md]: image does not show on raw.githack so can you put relative path here?

Comment (MatthewMarroquin) [reply pages/vi/github-issues.md]: good catch I have put the relative path so now the image shows on raw.githack link

File: pages/vi/github-issues.md [hunk 2]
```diff
@@ -65,13 +65,13 @@ Now you can go and make the proposed changes to your local files. You can use an
 | --- |
 If you're still confused, that's quite alright. [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow) can be quite challenging at first. See this diagram below. For better context, the new branch you just made is inside your "Forked GitHub IO".
 
-## NEED DIAGRAM HERE
+![flowchart](../../images/flowchart.png)
```
Comment (dogi) [inline pages/vi/github-issues.md]: @Mattmtech please put the image in the folder of `pages/vi/images`

Comment (MatthewMarroquin) [reply pages/vi/github-issues.md]: @dogi image has been moved to `pages/vi/images` and the path in github-issues.md has been changed to reflect this change

**Review by dogi — APPROVED** [review]

**Review by lmmrssa — APPROVED** [review]

**Pull request merged**